apps/dashboard/app/views/batch_connect/sessions/card/_display_choices.html.erb

```diff
@@ -0,0 +1,7 @@
+<% user_context = session.user_context %>
+<% session.app.attributes.select(&:display?).map do |attribute| %>
+  <p>
+    <strong><%= attribute.label %>:</strong>
```

Comment:
I haven't pulled this down to see, but I would worry about whitespace here specifically, lack there of.

I supsect this renders
```markdown
key:value
```
instead of
```markdown
key: value
```

Though I'll have to pull this down to confirm which I'll do now.

Reply:
I see, it's because you had a newline there that there is a space. I would suggest putting them on the same line with a space - could just read better as that's how it'll display (it doesn't display as the next line with any `\n` newlines or breaks)

```ruby
<strong><%= attribute.label %>:</strong> <%= user_context.fetch(attribute.id, '') %>
```